staging/src/k8s.io/apiserver/pkg/admission/plugin/validatingadmissionpolicy/interface.go

```diff
@@ -71,6 +72,8 @@ type Matcher interface {
 	// BindingMatches says whether this policy definition matches the provided admission
 	// resource request
 	BindingMatches(a admission.Attributes, o admission.ObjectInterfaces, definition *v1alpha1.ValidatingAdmissionPolicyBinding) (bool, error)
+
+	GetNamespace(name string) (*corev1.Namespace, error)
```

Comment:
Can we doc what happens if the object is cluster-scoped (without a namespace) here?

Reply:
It is documented in https://github.com/kubernetes/kubernetes/blob/093afa03ba551d131b18cca84ae17e125e72ad44/staging/src/k8s.io/apiserver/pkg/admission/plugin/validatingadmissionpolicy/controller.go#L330. I will add more comments when I got my access back :)